src/app/content/highlights/components/ColorPicker.tsx

```diff
@@ -62,25 +62,24 @@ const ColorButton = styled(({className, size, style, ...props}: ColorButtonProps
 `;
 
 // tslint:disable-next-line:variable-name
-const ColorPicker = ({className, ...props}: Props) => {
-
+const ColorPicker = styled(({className, ...props}: Props) => {
```

Comment:
were any of the changes in this file necessary?

Reply:
no.